templates/index.html

```diff
@@ -37,6 +37,7 @@
                           <input id="survey-quantity" value="1" />
                         </div>
                         <input class="btn btn-default" type="submit" />
+                        <input class="btn btn-default" type="click" value="Validate" id="validate"/>
```

Comment:
In order to keep styling as per other button, this should be a button '<button class="btn btn-default" id="validate">Validate</button>'